README.md

```diff
@@ -163,6 +163,7 @@ go generate
   - Needed for testing OAUTHBEARER Application Principal
```

Comment:
Please add to Compatibility section that this Terraform Provider version only works with the latest API that returns owners for the schema correctly